### PR TITLE
reference-manual: linux: improve factory reset docs

### PR DESCRIPTION
**Pull request by bravl**
Add main distinction between full and partial factory reset. Add clarification that at this point in time RPMB is not touched during factory reset.

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

Improves the documentation for the factory reset.

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [x] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [x] Step through instructions (or ask someone to do so).
* [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [x] Match tone and style of page/section.
* [x] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [x] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [x] follow best practices for commits.
  * [x] Descriptive title written in the imperative.
  * [x] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [x] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 428e246678d5bfbaf72968a1d9cab6ea06c14244 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2047/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 67b07a78196e24a90051d4f2d51cd20d6522346f are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2048/docs/artifacts/html/index.html

File: source/reference-manual/linux/factory-device-reset.rst [hunk 1]
```diff
@@ -27,6 +27,7 @@ Partial factory reset
 ---------------------
 
 There are currently two options in partial reset.
+The main difference with full reset is that the device remains connected to the FoundriesFactory.
```
Comment (kprosise) [inline source/reference-manual/linux/factory-device-reset.rst:30]:
```suggestion
The main difference with full reset is that the device remains connected to the FoundriesFactory®.
```

**Comment by mwasilew** [comment]
I don't know if RPMB can be cleared without reflashing. The patch looks OK.

**Comment by bravl** [comment]
> I don't know if RPMB can be cleared without reflashing. The patch looks OK.

You can partially clear it without reflashing but handling of the rpmb is marked as todo in LmP at the moment.

**Pull request merged**